src/ray/gcs/gcs_server/default_job_info_access_handler.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
The google code specification does say that using `#ifndef, #define, #endif` to prevent duplicate definition of header files, and it also said to use `<PROJECT>_<PATH>_<FILE>_H` to describe the header file, but when we refactor the project, it is difficult to ensure that we also modify `#ifndef, #define, #endif` together, so I still think it is better to use `#pragma once`.
What do you think? @raulchen @ericl

Reply:
I think we have been using ifdef, so probably best to stay consistent with that.

Reply:
> I think we have been using ifdef, so probably best to stay consistent with that.

Okey, I fix it soon.